src/script/components/UserList.tsx

```diff
@@ -52,44 +52,38 @@ export enum UserlistMode {
 const USER_CHUNK_SIZE = 64;
 
 export interface UserListProps {
-  addToSelected: (user: User) => void;
   arrow: boolean;
   click: (userEntity: User, event: MouseEvent | KeyboardEvent) => void;
   conversation: Conversation;
   conversationRepository: ConversationRepository;
   conversationState?: ConversationState;
-  filter: string;
   highlightedUsers: () => User[];
   infos: Record<string, string>;
   maxVisibleUsers: number;
   mode: UserlistMode;
   noSelfInteraction: boolean;
   noUnderline: boolean;
+  observers: {filter: ko.Observable<string>; users: ko.ObservableArray<User>};
```

Comment:
Let's rather call it `observables` rather than `observers`. 
Also `selected` can be put in this `observables` object :)

Reply:
https://github.com/wireapp/wire-webapp/pull/12782/commits/3b2d685a2d98c9d3738fc1152b121df81e59f74a